### PR TITLE
 Localize message from when clear the chat.

### PR DESCRIPTION
**Pull request by Tainan404**
- Message "The public chat history was cleared by a moderator" now is able be translated.

### CODE REVIEW

File: bigbluebutton-html5/private/locales/en.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "app.chat.dropdown.save": "Save",
     "app.chat.label": "Chat",
     "app.chat.emptyLogLabel": "Chat log empty",
+    "app.chat.clearedChat": "The public chat history was cleared by a moderator",
```
Comment (OZhurbenko) [inline bigbluebutton-html5/private/locales/en.json]: clearPublicChatMessage

**Comment by OZhurbenko** [comment]
Added a custom message to the German locale, it seems to be working fine.

But it looks like the clear chat message lost its styles.

Before:
clear_chat_before

After:
clear_chat_after

Also, there is an issue with the "Copy" option.
This is what I got after copying the messages from the example above:

```
[16:17] <br>Welcome to <b>Demo Meeting</b>!<br><br>For help on using BigBlueButton see these (short) <a target="_blank" href="http://www.bigbluebutton.org/content/videos"><u>tutorial videos</u></a>.<br><br>To join the audio bridge click the headset icon (upper-left hand corner).  Use a headset to avoid causing background noise for others.<br><br><br>This server is running <a href="http://docs.bigbluebutton.org/" target="_blank"><u>BigBlueButton</u></a>.
[16:18] : PUBLIC_CHAT_CLEAR
```

Probably shouldn't see the name of the variable there.

**Comment by Tainan404** [comment]
@OZhurbenko, i noted such bug, but no found it an way of make in the service that export the messages. If have a idea of how make, i accept suggestion.

File: bigbluebutton-html5/imports/ui/components/chat/container.jsx [hunk 2]
```diff
@@ -56,24 +73,6 @@ export default injectIntl(createContainer(({ params, intl }) => {
 
       title = intl.formatMessage(intlMessages.titlePrivate, { 0: chatUser.name });
       chatName = chatUser.name;
-
-      if (!chatUser.isOnline) {
-        const time = Date.now();
-        const id = `partner-disconnected-${time}`;
-        const messagePartnerLoggedOut = {
```
Comment (OZhurbenko) [inline bigbluebutton-html5/imports/ui/components/chat/container.jsx]: I'm very confused now. Double-checked, and yeah.. we don't have the message "xxx left the meeting" anymore here.
It was silently removed here.

@antobinary confirm please, don't we need this message anymore?

notification_left_the_meeting

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/chat/container.jsx]: @OZhurbenko from what I know we still need this `X has left the meeting`.

**Comment by OZhurbenko** [comment]
Looks like it's working, but could you replace [this line](https://github.com/bigbluebutton/bigbluebutton/pull/4712/files#diff-f239508bf2b98e0c6448adf9b360a52cR84) with:

```
text: `<b><i>${intl.formatMessage(intlMessages.partnerDisconnected, { 0: chatUser.name })}</i></b>`,
```

It would be great to have some styles for the "user disconnected" message.

**Comment by oswaldoacauan** [comment]
@OZhurbenko bad idea to use html tags. If we want system messages to be bold and italic we need to make all of them the same style. We probably should ask @tylercopeland what he thinks.

Maybe dont have styles for the WelcomeMessage but bold+italic for everything else?

**Comment by capilkey** [comment]
The welcome message can have HTML tags in it. In the Flash client the only thing we set is the colour of the welcome message. People might want to emphasize certain parts of the message with bold. If you put the tags in the message then the view is simplified because it doesn't need to know the difference between a "system message" and a "user message".

**Comment by oswaldoacauan** [comment]
> The welcome message can have HTML tags in it. In the Flash client the only thing we set is the colour of the welcome message.

Yep, we dont style (other than the color) the welcome message as well.

> If you put the tags in the message then the view is simplified because it doesn't need to know the difference between a "system message" and a "user message".

But then we need to put them on the translation message which imo is not the ideal scenario.

**Comment by capilkey** [comment]
That's a good point. I hadn't considered what it would do to the translation strings. It's much nicer when views can be kept simple, but it seems unavoidable in this case I guess.

**Comment by OZhurbenko** [comment]
> bad idea to use html tags. 
@oswaldoacauan that's what we do to the "The public chat history cleared by a moderator on a screenshot above".

**Comment by oswaldoacauan** [comment]
@OZhurbenko yeah, I got confused since we have two identical lines in that code. @Tainan404 will send a commit with a fix.

**Comment by OZhurbenko** [comment]
@oswaldoacauan I mean - if there is a better way to make them look the same - let's do it differently, I'm just saying that we should be consistent with how they look.. and differ them from the standard messages' plain text somehow.

**Comment by oswaldoacauan** [comment]
@OZhurbenko and I agree, they should be bold and italic. I just misunderstood where you wanted to add the tags. :see_no_evil:

**Comment by antobinary** [comment]
fixes #4698

**Comment by OZhurbenko** [comment]
Frankly speaking I expected you just to replace one line, add a few tags around it.
You decided to "refactor" it, well.. now `X has left the meeting` doesn't get removed from the private chat. 

Just open a private chat with the user, and refresh that user's tab.

**Comment by Tainan404** [comment]
I could not reproduce the error.

message

**Comment by OZhurbenko** [comment]
@Tainan404 hmm, it's very strange, but you are right, it's working perfectly fine.
It didn't want to work for me 2 days ago for some reason.

Great work!

**Pull request merged**